docs/adding-to-project.md

```diff
@@ -1,9 +1,8 @@
 # Adding BLT to an existing project
 
-Adding BLT to an existing project can be much more complex than simply creating a new site with BLT. There are a few reasons for this:
+Adding BLT to an existing project can be more complex than simply creating a new site with BLT since BLT includes a project template and expects the files in your project to match the structure and contents of this template.
 
-1. BLT expects your project to have a particular directory structure and set of files.
-1. Your existing dependencies may conflict with BLT's dependencies.
```

Comment:
I removed the note about dependency conflicts because this is true for any Composer package and not unique to BLT. Plus, BLT has many fewer dependencies in the 10.x branch than it used to.